src/main/java/nextstep/member/MemberController.java

```diff
@@ -21,9 +24,9 @@ public ResponseEntity createMember(@RequestBody MemberRequest memberRequest) {
     }
 
     @GetMapping("/me")
-    public ResponseEntity me() {
-        Long id = 1L;
-        Member member = memberService.findById(id);
+    public ResponseEntity me(HttpServletRequest request) {
```

Comment:
`ResponseEntity` 클래스는 Generic으로 정의된 것으로 알고 있어요.
아래와 같이 어떤 타입을 가지고 있는 클래스인지를 명확하게 보여주는 것이 좋다고 생각합니다.
어떻게 생각하시나요 ~ ?

```suggestion
    public ResponseEntity<Member> me(HttpServletRequest request) {
```

Reply:
추가로 또 생각해보시면 좋을 것 같은 부분이, Entity 클래스를 Service, Controller, Repository까지 같이 사용하면 I/F만 변경되거나 DB 스키마가 변경되었을 때 다른 곳에 영향이 있지는 않을지도 고민해보시면 좋을 것 같습니다!